cypress/integration/login.js

```diff
@@ -1,30 +1,28 @@
+import LoginPage from '../objects/loginPage';   //.. - objects up
+import TimeLogging from '../objects/timeLogging';
 describe('Sourcebooks login', function() {

```

Comment:
Feels like this part could use an empty line to separate the imports from the describe block :)